pages/signup/register.tsx

```diff
@@ -23,12 +21,69 @@ import { getServerSession } from "next-auth/next";
 import Link from "next/link";
 import Head from "next/head";
 import { ErrorStatus } from "@components/forms/Alert/Alert";
+import { FormikHelpers } from "formik";
+import { fetchWithCsrfToken } from "@lib/hooks/auth/fetchWithCsrfToken";
+import { hasError } from "@lib/hasError";
+import { logMessage } from "@lib/logger";
+import { Verify } from "@components/auth/Verify";
+import { useLogin } from "@lib/hooks/auth";
 
 const Register = () => {
   const { isLoading, status: registrationOpen } = useFlag("accountRegistration");
-  const { username, password, needsConfirmation, register, authErrorsState, authErrorsReset } =
-    useRegister();
   const { t } = useTranslation(["signup", "common"]);
+  const {
+    username,
+    password,
+    authenticationFlowToken,
+    needsVerification,
+    setNeedsVerification,
+    authErrorsState,
+    authErrorsReset,
+    login,
+    handleErrorById,
+  } = useLogin();
+
+  const register = async (
+    {
+      username,
+      password,
+      name,
+    }: {
+      username: string;
+      password: string;
+      name: string;
+    },
+    { setSubmitting }: FormikHelpers<{ username: string; password: string; name: string }>
+  ) => {
+    authErrorsReset();
+    try {
+      // Register the user. An error is thrown if the username exists etc.
+      await fetchWithCsrfToken("/api/account/register", {
+        username,
+        password,
+        name,
+      });
+
+      // TEMP - to remove once confirmation code removed
+      // Add a Breakpoint below, to give you time first confirm the account in cognito before login
+      logMessage.info("");
```

Comment:
Can remove this :)

Reply:
True :) The problem though is without this there is no way to test the 2FA flow after the register step. We still need a delay to allow us to go into Cognito and manually confirm an account before the code moves on to the auto-login step.

But if we'd rather get the code ready for future 2FA testing when the manual confirm step is no longer needed, then it should be removed?

Reply:
@bryan-robitaille we're looking to get setup to test without manually confirming correct?

Reply:
Sounds good, will remove :)